### ISSUE TITLE
change/delete inside/around enhancements list

### ISSUE THREAD

**Issue by rhelsing** [guillermooo/Six#125]
* Implement around. Currently inside is supported
* Multiline
```
{
  key: value //from normal mode, ci{ should work
}
```

* Implement t (tag) modifier - (cit, cat, dit, dat)
```
<div>text</div>
```

**Comment by guillermooo** [comment]
### To-do Multiline

- [ ] **(**
- [ ] **[**
- [ ] **{**

As far as I can tell, for the rest of the delimiters Vim does not perform a multiline search.

**Comment by rhelsing** [comment]
@guillermooo 

![screen shot 2017-06-25 at 10 16 14 am](https://user-images.githubusercontent.com/392594/27518214-5f390018-598f-11e7-85b7-059d79e15437.png)

From normal mode:
```
cit
```

result:
![screen shot 2017-06-25 at 10 16 22 am](https://user-images.githubusercontent.com/392594/27518217-740e722a-598f-11e7-954d-4dff93324c18.png)

**Comment by rhelsing** [comment]
@guillermooo I know this isn't core vim, but worth considering a path to implementing the following, either as a plugin (open to be developed by someone else) or as core:

from vim-rails

![screen shot 2017-06-25 at 10 22 11 am](https://user-images.githubusercontent.com/392594/27518247-4aad337a-5990-11e7-8218-dfc869bf5e8d.png)

https://www.cheatography.com/blackxored/cheat-sheets/vim-rails/

**Comment by guillermooo** [comment]
If it was easy to implement the vim plugin api, that would be ideal, but I'm not sure right now how much work that would be. Don´t expect it any time soon, but as a long term goal, however, it sounds reasonable to at least try.

A quicker solution would to endow Six with its own plugin system. That sort of thing already existed in Vintageous and it worked okay. That is planned for Six, but without a timeline. However, once I'm done cleaning up a few things, it´s possible that all the required moving parts are all already there and enabling external plugins would be a matter of exposing a couple of hooks.

In short, I think there will be ways to add the functionality that you want added to Six be means of plugins.

**Comment by guillermooo** [comment]
Status update: Missing support for tags.

**Comment by rhelsing** [comment]
@guillermooo amazing work! love it so much!

**Comment by guillermooo** [comment]
Further fixes to visual mode coming soon! 😉

**Comment by guillermooo** [comment]
I'll leave polish for when the caret is inside tags/at tag boundaries for a bit later.
